<?xml version="1.0" ?>
<launch>
  <arg name="log_level" default="1"/>

  <arg name="mqtt_bridge_ip" default="$(optenv MQTT_BRIDGE_IP localhost)" doc="ip of bridge"/>
  <arg name="mqtt_bridge_port" default="$(optenv MQTT_BRIDGE_PORT 8883)" doc="port of bridge"/>

  <!-- Launch Robot -->
  <include file="$(find turtlebot_gazebo)/launch/turtlebot_world.launch">
    <arg name="world_file" value="$(find vrviz_ros)/maps/object-search-training.world" />
    <arg name="gui" value="false"/>
  </include>
  <include file="$(find turtlebot_gazebo)/launch/amcl_demo.launch">
    <arg name="map_file" value="$(find vrviz_ros)/maps/object-search-arena.yaml" />
  </include>

  <!-- Begin exploratory navigation -->
  <node pkg="vrviz_ros" name="robot_exploration" type="robot_exploration.py" output="screen">
        <param name="log_level" value="$(arg log_level)" type="int"/>
  </node>

</launch>